pkg/endpoint/public_ip.go

```diff
@@ -212,6 +214,12 @@ func publicDNSIP(clientset kubernetes.Interface, namespace, fqdn string) (string
 		return "", errors.Wrapf(err, "error resolving DNS hostname %q for public IP", fqdn)
 	}
 
+	if len(ips) > 1 {
+		sort.Slice(ips, func(i, j int) bool {
```

Comment:
I think we have to understand the reason for multiple IPs on AWS (in OCP4.12 and not in OCP4.11) and if we can successfully connect with ALL the IPs returned by DNS query. If so, this approach should work.